### PR TITLE
Add tslint (issue #6)

### PR DESCRIPTION
**Pull request by jotak**
Copied more or less the rules from kiali's tslint
Up to relax some rules when they're annoying for development

### CODE REVIEW

File: web/react/tslint.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "extends": ["tslint-react", "tslint-no-circular-imports"],
+  "defaultSeverity": "warning",
```
Comment (israel-hdez) [inline web/react/tslint.json]: Perhaps you want to remove this line?
Not sure if default severity is "error". I guess the linter should make fail CI and probably this line will make it pass with just warnings. Although, also I'm not sure if this is intentional for now.

Comment (jotak) [reply web/react/tslint.json]: done .. I took that from kiali-ui but you're right we can be stricter

Comment (israel-hdez) [reply web/react/tslint.json]: In kiali-ui, Travis uses https://github.com/kiali/kiali-ui/blob/master/tslint.prod.json... That way, while developing, `yarn start` works with warnings, but Travis fails with errors :)

Comment (jotak) [reply web/react/tslint.json]: ah ok, fair enough!

**Pull request merged**